docs/team/ytingtan.md

```diff
@@ -14,15 +14,10 @@ Given below are my contributions to the project.
     * What it does: 
       * Indicates which stage of application the candidate is at. 
       * As an applicant tracking system, a crucial feature is to enable recruiters to view the application stage of each candidate at a glance.
-      * Some example statuses include:
-        - Application Received
-        - OA in Progress
-        - Shortlisted for Interview
-        - Accepted
-        - Rejected
+      * Some example statuses include: Application Received, OA in Progress, Shortlisted for Interview, Accepted, Rejected
       * It is a compulsory attribute of a candidate, as every candidate has an application status. 
       * The Status attribute can be used in two ways: 
-        1. Adding a new candidate `add n/John Doe e/john@mail.com p/10384280 s/Application Received` adds a new candidate with name `John Doe`, email `john@mail.com`, phone number `10384280`, and status of `Application Received`.
+        1. Adding a new candidate `add n/John Doe p/10384280 e/john@mail.com s/Application Received` adds a new candidate with name `John Doe`, phone number `10384280`, email `john@mail.com`, and status of `Application Received`.
```

Comment:
You might have to rewrite this example with all the compulsory fields for the add command mentioned in it. For example: address is missing.